src/bezpath.rs

```diff
@@ -156,10 +161,12 @@ impl BezPath {
     /// the square of the distance to the point.
     pub fn nearest(&self, p: Vec2, accuracy: f64) -> (usize, f64, f64) {
         let mut best = None;
-        for (ix, seg) in self.segments() {
-            let (t, r) = seg.nearest(p, accuracy);
-            if best.map(|(_, _, r_best)| r < r_best).unwrap_or(true) {
-                best = Some((ix, t, r));
+        for ix in 0..self.0.len() {
```

Comment:
I think I prefer the previous state (before afcb951) if the comment is fixed to indicate that it returns the segment index.